TransportSecurity.asciidoc

```diff
@@ -0,0 +1,81 @@
+== Server Transport Security
+
+JGroups offers various protocols to secure the transport, however they require complex setup which could be simplified by the server security configuration.
+
+== Encryption
+
+=== SSL/TLS
+
+JGroups does not directly offer the use of TLS/SSL for the TCP transport, although it does allow setting a custom `SocketFactory`.
+
+By using the server identity of a security realm, we can supply JGroups with an SSL-enabled SocketFactory. The server identity must have both a keystore as well as a truststore so that certificates are mutually trusted.
+
+By using namespaces, we can extend the transport configuration as follows:
+
+[source,xml]
+----
+<cache-container>
+    <transport>
+        <server:encryption>
```

Comment:
I would rather create an `<encrypt>` element under `<stack>` since this is related to JGroups. And also I could take advantage of it for xsite :)